.github/pull_request_template.md

```diff
@@ -1,12 +1,21 @@
 #### Trac ticket number
-<!-- Replace XXXXX with the corresponding Trac ticket number, or delete the line and write "N/A" if this is a trivial PR. -->
+<!-- Replace XXXXX with the corresponding Trac ticket number. -->
+<!-- Or delete the line and write "N/A" if this is a trivial PR. -->
 
 ticket-XXXXX
 
 #### Branch description
 Provide a concise overview of the issue or rationale behind the proposed changes.
 
+#### AI Assistance Disclosure (REQUIRED)
+<!-- Please select exactly ONE of the following: -->
+- [ ] **No AI tools were used** in preparing this PR.
+- [ ] **AI tools were used**, and I have disclosed which ones, and fully reviewed and verified their output.
```

Comment:
```suggestion
Provide a concise overview of the issue or rationale behind the proposed changes.
If you have used AI tools to help with this contribution, it is useful to say what for.
```

Personally, I would let them write this info in the branch description. I also think it's kinda optional, if the contribution is good and follows the guidelines, it doesn't matter to me.